packages/core/src/FormField/FormField.stories.js

```diff
@@ -92,9 +99,9 @@ storiesOf('FormField', module)
         id='valid'
         name='valid'
         placeholder='hello@example.com'
-        color='green'
+        color='secondary'
       />
-      <Icon name='Success' color='green' />
+      <SuccessIcon color='secondary' />
```

Comment:
Could you change this to `success` instead of `secondary` please

Reply:
Yup for sure, updated the input too!